cfme/tests/v2v/test_v2v_migrations_ui.py

```diff
@@ -322,6 +322,33 @@ def _cleanup():
         pass
 
 
+@pytest.fixture
+def setup_user_for_v2v_migration(appliance, new_credential):
+    """
+    Fixture to set up user for v2v migration.
+
+    A new role is also created that initially  has access to all product features. The newly created
+    user has this role.
+
+    As part of the test, the role is updated so that the Migration tab is unavailable  in the UI to
+    users with this role.
+    """
+    role = new_role(appliance=appliance,
+                    product_features=[(['Everything'], True)])
+    group = new_group(appliance=appliance, role=role.name)
+    user = new_user(appliance=appliance, group=group, credential=new_credential)
+
+    yield user, role
+
+    user.delete_if_exists()
+    group.delete_if_exists()
+    role.delete_if_exists()
+
+
+@pytest.mark.parametrize('migration_feature_availability_for_role', ['disabled', 'enabled'],
+    ids=['disabled', 'enabled'],
```

Comment:
optional: These are default `ids` as we are iterating over the same list `['disabled', 'enabled']`

Reply:
Done